### PR TITLE
showSimplifiedLoginWidget function

### PR DESCRIPTION
**Pull request by marjuszkiewicz**
Will load simplified login widget, and display it on page

### CODE REVIEW

File: src/identity.js [hunk 1]
```diff
@@ -927,6 +927,55 @@ export class Identity extends EventEmitter {
...
+     * Function responsible for loading and displaying simplified login widget. How often
+     * widget will be display is up to you. Preferred way would be to show it once per user,
+     * and store that info in localStorage. Widget will be display only if user is logged in to SSO.
+     *
+     * @param {object} loginParams - the same as `options` param for login function. Login will be called on user
+     * continue action
+     * @return {Promise<boolean|SDKError>} - will resolve to true if widget will be display. Otherwise will throw SDKError
+     */
+    async showSimplifiedLoginWidget(loginParams) {
+        // getUserContextData doenst throw exception
```
Comment (poulch) [inline src/identity.js]: `doesn't`

File: src/identity.js [hunk 2]
```diff
@@ -927,6 +927,55 @@ export class Identity extends EventEmitter {
...
+        const userData = await this.getUserContextData();
+        const widgetUrl = '//localhost:8080/main.js'; // this._bffService.makeUrl('simplified-login-widget', { client_id: this.clientId }, false);
```
Comment (poulch) [inline src/identity.js]: Remove localhost

File: src/identity.js [hunk 3]
```diff
@@ -927,6 +927,55 @@ export class Identity extends EventEmitter {
...
+
+                const initialParams = {
+                    display_text: userData.display_text,
+                    env: this.env,
+                    client_name: userData.client_name,
+                    clientId: this.clientId,
+                    windowWidth: () => window.innerWidth,
+                    windowOnResize: (f) => {
+                        window.onresize = f;
+                    },
```
Comment (poulch) [inline src/identity.js]: Let make all params in camelcase

File: src/identity.js [hunk 4]
```diff
@@ -927,6 +927,55 @@ export class Identity extends EventEmitter {
...
+                    this.login(Object.assign(loginParams, {loginHint: userData.identifier}));
+                };
+
+                const simplifiedLoginWidget = document.createElement("script");
+                simplifiedLoginWidget.type = "text/javascript";
+                simplifiedLoginWidget.src = widgetUrl;
+                simplifiedLoginWidget.onload = () => {
+                    window.openSimplifiedLoginWidget(initialParams, loginHandler);
+                    resolve(true);
+                };
```
Comment (poulch) [inline src/identity.js]: When we open simplified widget each time new script tag will be append into body, in my option we should prepare logic to not load widget script second time

File: src/identity.js (hunk 1 above)
Comment (marjuszkiewicz) [reply src/identity.js]: 👍

File: src/identity.js (hunk 2 above)
Comment (marjuszkiewicz) [reply src/identity.js]: good catch!

File: src/identity.js (hunk 4 above)
Comment (marjuszkiewicz) [reply src/identity.js]: Done

**Pull request merged**